nx/include/switch/services/capssc.h

```diff
@@ -23,12 +24,22 @@ Service* capsscGetServiceSession(void);
  * @note buffer_index and buffer_count correspond to buffers with size 0x384000(1280*720*4). These must not be negative.
  * @param buf Output buffer containing the RGBA8 image.
  * @param size Size of buf, should be 0x384000(1280*720*4) * buffer_count.
- * @param layer_stack Value 0 can be used for this.
+ * @param layer_stack See \ref ViLayerStack.
  * @param width Image width, must be 1280.
  * @param height Image height, must be 720.
  * @param buffer_count Total number of output image buffers.
  * @param buffer_index Starting image buffer index. Must be < buffer_count.
  * @param timeout Timeout in nanoseconds. A default value of 100000000 can be used.
  */
-Result capsscCaptureRawImageWithTimeout(void* buf, size_t size, u32 layer_stack, u64 width, u64 height, s64 buffer_count, s64 buffer_index, u64 timeout);
+Result capsscCaptureRawImageWithTimeout(void* buf, size_t size, ViLayerStack layer_stack, u64 width, u64 height, s64 buffer_count, s64 buffer_index, s64 timeout);
 
+/**
+ * @brief This takes a screenshot, with the screenshot being written as jpeg into the ouput buffer.
```

Comment:
ouput -> output

Reply:
FIY you can use github suggestions to fix these errors yourself